torchvision/prototype/transforms/_geometry.py

```diff
@@ -202,15 +201,12 @@ def __init__(self, size: Union[int, Sequence[int]], vertical_flip: bool = False)
         self.size = _setup_size(size, error_msg="Please provide only two dimensions (h, w) for size.")
         self.vertical_flip = vertical_flip
 
-    def _transform(
-        self, inpt: Union[features.ImageType, features.VideoType], params: Dict[str, Any]
-    ) -> Union[List[features.ImageTypeJIT], List[features.VideoTypeJIT]]:
-        return F.ten_crop(inpt, self.size, vertical_flip=self.vertical_flip)
-
-    def forward(self, *inputs: Any) -> Any:
-        if has_any(inputs, features.BoundingBox, features.Mask):
+    def _check(self, sample: Any) -> None:
+        if has_any(sample, features.BoundingBox, features.Mask):
             raise TypeError(f"BoundingBox'es and Mask's are not supported by {type(self).__name__}()")
-        return super().forward(*inputs)
+
+    def _transform(self, inpt: Any, params: Dict[str, Any]) -> Any:
```

Comment:
Is this change intentional or the result of copy-paste?
```suggestion
    def _transform(
        self, inpt: Union[features.ImageType, features.VideoType], params: Dict[str, Any]
    ) -> Union[List[features.ImageTypeJIT], List[features.VideoTypeJIT]]:
```

Reply:
The latter. Happened during fixing merge conflicts. Thanks for pointing it out!